asus/zephyrus/ga402x/shared.nix

```diff
@@ -32,6 +32,12 @@ in {
     keyboard.autosuspend.enable = (
       mkEnableOption "Enable auto-suspend on the internal USB keyboard (ASUS N-KEY Device) on Zephyrus GA402X"
     ) // { default = defaultAutosuspendEnable; };
+    # The ASUS 8295 ITE device will cause an immediate wake-up when trying to suspend the laptop.
+    # After the first successful hibernate, it will work as expected, however.
+    # NOTE: I'm not actually sure what this device, as neither the touchpad nor the M1-M4 keys cause a wake-up.
+    ite-device.wakeup.enable = (
+      mkEnableOption "Enable power wakeup on the internal USB keyboard-like device (8295 ITE Device) on Zephyrus GA402X"
+    ) // { default = false; };
```

Comment:
`mkEnableOption` already defaults to false, right?

Reply:
It does.

Reply:
https://github.com/NixOS/nixos-hardware/pull/1156